docs/tutorials/reproducing_pober_2014.ipynb

```diff
@@ -0,0 +1,721 @@
+{
```

Comment:
<p>It does seem good.  For a check, the beam was defined as a Gaussian with a sigma of sigma = 0.0643 at 150 MHz</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/steven-murray/21cmSense/pull/82/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>